XNAControls/XNACheckBox.cs

```diff
@@ -276,44 +262,45 @@ public override void Draw(GameTime gameTime)
 
         if (!string.IsNullOrEmpty(Text))
         {
-            Color textColor;
-            if (!AllowChecking)
-                textColor = Color.Gray;
-            else
-                textColor = IsActive ? HighlightColor : IdleColor;
-
-            DrawStringWithShadow(Text, FontIndex,
-                new Vector2(checkedTexture.Width + TextPadding, textYPosition),
-                textColor, 1.0f, UISettings.ActiveSettings.TextShadowDistance);
+            Color textColor = !AllowChecking ? Color.Gray : IsActive ? HighlightColor : IdleColor;
+            DrawStringWithShadow(
+                Text,
+                FontIndex,
+                new(checkedTexture.Width + TextPadding, textYPosition),
+                textColor,
+                1.0f,
+                UISettings.ActiveSettings.TextShadowDistance);
         }
 
         // Might not be worth it to save one draw-call per frame with a confusing
         // if-else routine, but oh well
         if (checkedAlpha == 0.0)
         {
-            DrawTexture(clearTexture,
-                new Rectangle(0, checkBoxYPosition,
-                clearTexture.Width, clearTexture.Height), Color.White);
+            DrawTexture(
+                clearTexture,
+                new Rectangle(0, checkBoxYPosition, clearTexture.Width, clearTexture.Height),
+                Color.White);
```

Comment:
Unless the overall line length gets inconveniently wide, keep the first parameter on the same line with the function call.